UPGRADING.md

```diff
@@ -56,10 +57,16 @@ the installation process and run both schema and data migrations.
  sudo su - postgres -c "psql -c \"alter user round password 'round'\""
  sudo su - postgres -c "psql roundware -c 'create extension postgis'"
  ```
-6. Run migration required for Postgres update: `python manage.py migrate`
-7. Import dumped db from MySQL into Postgres db: `python manage.py loaddata rw-pre-postgres.json`
-8. Update to most recent code including speaker polygon updates: `git checkout develop`
-9. Deploy newest code (includes pip installs and db migrations): `sudo ./deploy.sh`
+7. Deploy code again to run pip installs and minor migration in preparation for data import: `sudo ./deploy.sh`
+8. Prepare database and import fixture
+ 1. switch to `roundware` user in order to run `manage.py` commands: `sudo su roundware`
+ 2. enter virtualenv: `source /var/www/roundware/bin/activate`
+ 3. set PYTHONPATH: `export PYTHONPATH=/var/www/roundware/source/`
```

Comment:
These should be set in the bash profile for the roundware user. No reason to do it every time. Doesn't need to be fixed this PR.